sdk/lib/crt/stdlib/rot.c

```diff
@@ -19,8 +19,16 @@
 #pragma function(_rotr, _rotl, _rotr, _lrotl, _lrotr)
 #endif
 
-unsigned int _rotr( unsigned int value, int shift );
-unsigned long _lrotr(unsigned long value, int shift);
+#if defined (__clang__) && !defined(_MSC_VER)
+#define ASM_ALIAS __asm__
+#else
+#define ASM_ALIAS(x)
+#endif
+
+unsigned int _rotr( unsigned int value, int shift ) ASM_ALIAS("__rotr");
```

Comment:
I'm not clear on why this is needed. Isn't `__rotr` the default cdecl name for a C function called `_rotr`?

Reply:
those functions are builtins in CLang, and it rejects redefinig them. Look up in the file, _rotr is defined as __function_rotr.
Defining there symbol name with an ASM alias is fine, though.

Reply:
Oh. That makes sense, thanks.
Alternatively, perhaps `-fno-builtin=_rotr` could make this more obvious?

Also, is this okay for x64, or do we need to remove the extra underscore there?

Reply:
I don't know, but I guess this will have to be fixed.